### ISSUE TITLE
Change Password Form Color/Messages

### ISSUE THREAD

**Issue by nicklitvin** [ocf/ocfstatic#282]
I think the "Required. Your current password" messages in "/dashboard/profile" below the text inputs are a little redundant since upon submission the user is notified if any of the inputs are left blank (or password doesn't meet requirements in future maybe). 

Another concern with the text is the color choice. Whenever a text input is clicked on and then clicked off and the input remains blank, the text changes to a color that is very similar to the background making it difficult to read. 

Another relating concern is that when these messages are removed completely from the .Vue file, they are replaced with a "Please fill out this field" text when the original message would have otherwise changed from white to red when the input is left blank. This function and message originates from a different file and might be worth adjusting.
